NIPTool/crud/update.py

```diff
@@ -1,12 +1,12 @@
 import logging
 
 from NIPTool.adapter import NiptAdapter
-from NIPTool.models.database import Sample
+from NIPTool.models.database import DataBaseSample
 
 LOG = logging.getLogger(__name__)
 
 
-def sample(adapter: NiptAdapter, sample: Sample) -> dict:
+def sample(adapter: NiptAdapter, sample: DataBaseSample) -> dict:
```

Comment:
Should not this return a pydantic object?

Reply:
it shouldn't return anything actually! :)

Reply:
Aha but the return type should be `None`?

Reply:
!! Yes :)